content/intuitionistic-logic/soundness-completeness/lindenbaum.tex

```diff
@@ -86,6 +86,9 @@
   \olref{defn:prime1}, \olref{defn:prime2}, and~\olref{defn:prime3} of
   \olref{defn:prime}.
 
+  I dont get the following three paragraphs. Maybe (1), (2), (3) is
+  proved in the wrong order?
```

Comment:
I think it's correct if the last (1) is changed to (2). Thanks for spotting that!